e2e/test/scenarios/question/notebook-data-source.cy.spec.ts

```diff
@@ -393,6 +393,57 @@ describe("scenarios > notebook > data source", { tags: "@OSS" }, () => {
   });
 });
 
+describe("issue 28106", () => {
+  beforeEach(() => {
+    const dialect = "postgres";
+
+    resetTestTable({ type: dialect, table: "many_schemas" });
+    restore(`${dialect}-writable`);
+    cy.signInAsAdmin();
+
+    resyncDatabase({ dbId: WRITABLE_DB_ID });
+  });
+
+  it(
+    "should not jump to the top of schema list when scrolling (metabase#28106)",
+    { tags: "@external" },
+    () => {
+      startNewQuestion();
+      entityPickerModal().within(() => {
+        entityPickerModalTab("Tables").click();
+        cy.findByText("Writable Postgres12").click();
+
+        entityPickerModalLevel(1)
+          .findByTestId("scroll-container")
+          .as("schemasList");
+
+        // the list is virtualized and the scrollbar height changes during scrolling (metabase#44966)
+        // that's why we need to scroll twice and wait
+        cy.get("@schemasList").scrollTo("bottom");
+        cy.wait(100);
+        cy.get("@schemasList").scrollTo("bottom");
+
+        // assert scrolling worked and the last item is visible
+        entityPickerModalItem(1, "Public").should("be.visible");
+
+        // simulate scrolling up using mouse wheel 3 times
+        cy.get("@schemasList").realMouseWheel({ deltaY: -100 });
+        cy.wait(100);
+        cy.get("@schemasList").realMouseWheel({ deltaY: -100 });
+        cy.wait(100);
+        cy.get("@schemasList").realMouseWheel({ deltaY: -100 });
+        cy.wait(100);
+
+        // assert first item does not exist - this means the list has not been scrolled to the top
+        cy.findByText("Domestic").should("not.exist");
```

Comment:
Is this actually removed from the DOM or only not visible?

Reply:
Removed because because the list is virtualized.